src/test/groovy/org/mockito/release/notes/vcs/DefaultCommitApproverTest.groovy

```diff
@@ -0,0 +1,52 @@
+package org.mockito.release.notes.vcs
+
+import spock.lang.Specification
+
+class DefaultCommitApproverTest extends Specification {
```

Comment:
This test covers implementation of IgnoreCiSkip and IgnoreCommitByConfigMessagePostfix. I would cover it in dedicated tests for these classes and here I would mock all predicates and check if method isTrue() works correctly for empty list of predicates, for predicates that are all false, all true, some of them false some true.